src/Api/Controllers/AccountsController.cs

```diff
@@ -60,11 +62,13 @@ public class AccountsController : Controller
         _userService = userService;
         _sendRepository = sendRepository;
         _sendService = sendService;
+        _captchaValidationService = captchaValidationService;
     }
 
     #region DEPRECATED (Moved to Identity Service)
 
-    [Obsolete("2022-01-12 Moved to Identity, left for backwards compatability with older clients")]
+    // This method is still used by self hosted intalls
```

Comment:
It's actually used for all clients unless they run in dev since we needed to maintain backwards compatibility. I've created a PR to resolve this though since sufficient time has elapsed.